test/sync_test.go

```diff
@@ -250,6 +253,161 @@ func testSyncCluster(t *testing.T) {
 	}
 }
 
+func testSyncClusterCreatePullAndDeleteEvents(t *testing.T) {
+	/*
+
+		This tests syncing files back and forth between three cluster members.
+		Their configs are in h1, h2 and h3. The folder "default" is shared
+		between all and stored in s1, s2 and s3 respectively.
+
+		Another folder is shared between 1 and 2 only, in s12-1 and s12-2. A
+		third folders is shared between 2 and 3, in s23-2 and s23-3.
+
+		During this test, we create 1K files remove then and create them again.
+		However, during these operations we will perform scan operations such
+		that other nodes will retrieve these options will data is changing.
```

Comment:
`third **folders** is shared`
`1K files remove **then**``
`**will** data is changing`